### DISCUSSION TITLE
Getting confuse while getting Eth price

### DISCUSSION THREAD

**Discussion by talhaEth** [Cyfrin/foundry-full-course-cu#796] [Q&A] [answered]
Can anybody explain me this
```
From where we are getting the Eth price ($2000)
```
![33333333](https://github.com/Cyfrin/foundry-full-course-f23/assets/66763322/966d30db-29f8-4699-bb54-7e7c8bb55c39)

```
here is the test case
```
![yyyy](https://github.com/Cyfrin/foundry-full-course-f23/assets/66763322/d862b65b-fed2-46dd-836c-5fb088b01a04)

**Comment by theirrationalone** [comment]
We are getting from it from getUsdValue -> our mockV3Aggregator from helper config Initial answer, decimals.

**Reply by talhaEth** [reply]
@theirrationalone 
is this price is fluctuate like real ETH value or this price will be the same all the time $2000 because this is mock price ?

**Reply by theirrationalone** [reply, marked as answer]
Yes It will be persistent until you change it by on your own. For the sake of curiosity, You can experiment with it though. 😉

**Reply by talhaEth** [reply]
Ahh... got it... we are passing $2000 value in our helperConfig file
thank you man
stay blessed !!

**Reply by theirrationalone** [reply]
You're Welcome @talha073 !, Glad to work with u. ;`) 
❤
